README.md

```diff
@@ -41,17 +41,17 @@ Quick start
 
 Test
 ----
-
-Create env file with following data:
-POSTGRES_DB=fill_value
-POSTGRES_USER=file_value
-POSTGRES_PASSWORD=fill_vale
-POSTGRES_HOST=fill
-POSTGRES_PORT=5435
-
+- Create .env file with following data (feel free to change as rquired):
```

Comment:
the .env file is usually a key=value format, rather than a shell file. You can export kv with
```shell
    while read -r line; do
        if [[ -n "$line" ]] && [[ $line != \#* ]]; then
            export "$line";
        fi;
    done < $1
```

Reply:
While I don't necessarily favour the current `.env` format, it's a long-standing convention that should not be ignored lightly given the potential effects on other parts of the system.

Reply:
@labstersteve the issue with shell files that you need the `shell` environment to use it. Thus `.env` because non-usable in other environment like IDEs that support k=v and .env, docker that supports k=v and others

Reply:
@akoidan by looking at the LTI repo, I see that the pattern for settings is:
1. provide sane defaults in various locations, e.g. `docker\settings\*`, `django_project\code\settings*`
2. provide the developer with ability to override settings from ad 1, by means of `.env` file
3. require developer to provide some human-specific settings in the `.env` file, like AWS keys

Does my observation match with your experience?

Reply:
also, since .env has to be k=v file, without sh instructions, this piece from the readme seems to be pointless, so removing: 
 `source .env` (before running tox)